Cargo.lock

```diff
@@ -362,6 +362,18 @@ dependencies = [
  "subtle 2.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "delg_cred_cdd"
+version = "0.1.0"
+dependencies = [
+ "amcl_wrapper 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)",
+ "failure 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)",
+ "log 0.4.8 (registry+https://github.com/rust-lang/crates.io-index)",
+ "rand 0.6.5 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I'd like to see you use `rand 0.7`.
Remove `serde_derive`. Use `serde = { version = "1.0", features = ["derive"] }